src/olympia/access/acl.py

```diff
@@ -71,9 +72,9 @@ def check_collection_ownership(request, collection, require_owner=False):
     if not request.user.is_authenticated():
         return False
 
-    if action_allowed(request, 'Admin', '%'):
+    if permissions.ADMIN.has_permission(request):
         return True
-    elif action_allowed(request, 'Collections', 'Edit'):
+    elif permissions.COLLECTIONS_EDIT.has_permission(request):
```

Comment:
Just thinking loud, I think I would like an API that looks like

```python
from olympia.access import perms # or `permissions as perms` as a general guideline

if perms.has_permission(request, perms.COLLECTIONS_EDIT):
    # ...
```

or generally how `action_allowed` looked like. It's just a feeling though, I can't explain what I dislike about it.